src/handlers/jumpToDefinitionHandler.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as fse from "fs-extra";
+import * as path from "path";
+import * as vscode from "vscode";
+import { Dependency } from "../explorer/model/Dependency";
+import { getMavenLocalRepository } from "../utils/contextUtils";
+import { UserError } from "../utils/errorUtils";
+import { ElementNode, getNodesByTag, XmlTagName } from "../utils/lexerUtils";
+
+export async function jumpToDefinitionHandler(node?: Dependency): Promise<void> {
+    if (node === undefined) {
+        throw new Error("Only Dependency can jump to definition.");
+    }
+
+    let selectedPath: string;
+    if (node.parent === undefined) {
+        selectedPath = node.projectPomPath;
+    } else {
+        const parent: Dependency = <Dependency> node.parent;
+        selectedPath = localPomPath(parent.groupId, parent.artifactId, parent.version);
+    }
+    await jumpToDefinition(selectedPath, node.artifactId);
+}
+
+async function jumpToDefinition(pomPath: string, aid: string): Promise<void> {
+    const contentBuf: Buffer = await fse.readFile(pomPath);
+    const projectNodes: ElementNode[] = getNodesByTag(contentBuf.toString(), XmlTagName.Project);
+    if (projectNodes === undefined || projectNodes.length !== 1) {
+        throw new UserError("Only support POM file with single <project> node.");
+    }
+
+    const projectNode: ElementNode = projectNodes[0];
+    const dependenciesNode: ElementNode | undefined = projectNode.children?.find(node => node.tag === XmlTagName.Dependencies);
+    const dependencyNode: ElementNode | undefined = dependenciesNode?.children && dependenciesNode?.children.find(node =>
```

Comment:
`dependenciesNode?.children && dependenciesNode?.children.find()` can be shorten to `dependenciesNode?.children?.find`